data_storage/export_format.md

```diff
@@ -5,7 +5,7 @@ Export Format
   working directory, with the file name `<profile_name>.tox`
 
 - **5.1.2** ![](/badge/req.png) If the client supports user avatars, they will be stored
-  under the `avatars/` directory in the root of the client's working directory.
+  under the `avatars/` directory in the root of the export directory.
```

Comment:
Also, the directory tree structure at the very end of this file shows that avatars are exported into "client working dir". I suppose that means that "export directory" is actually "client working directory". If that's the case, your PR is pointless and adds a lot of confusion.